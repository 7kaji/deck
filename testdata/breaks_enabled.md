---
breaks: true
---

# Testing Soft Line Breaks

This is a test
with soft line break
that should render as breaks.

---

## List with soft breaks

- First item with
  soft break
- Second item with
  another soft break

---

## Blockquote with breaks

> This is a quote
> with soft line breaks
> in multiple places.
