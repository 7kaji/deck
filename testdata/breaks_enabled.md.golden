[
  {
    "layout": "",
    "titles": [
      "Testing Soft Line Breaks"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "This is a test\n"
              },
              {
                "value": "with soft line break\n"
              },
              {
                "value": "that should render as breaks."
              }
            ]
          }
        ]
      }
    ],
    "headings": {
      "1": [
        "Testing Soft Line Breaks"
      ]
    }
  },
  {
    "layout": "",
    "titles": [
      "List with soft breaks"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "First item with\n"
              },
              {
                "value": "soft break"
              }
            ],
            "bullet": "-"
          },
          {
            "fragments": [
              {
                "value": "Second item with\n"
              },
              {
                "value": "another soft break"
              }
            ],
            "bullet": "-"
          }
        ]
      }
    ],
    "headings": {
      "2": [
        "List with soft breaks"
      ]
    }
  },
  {
    "layout": "",
    "titles": [
      "Blockquote with breaks"
    ],
    "block_quotes": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "This is a quote\n"
              },
              {
                "value": "with soft line breaks\n"
              },
              {
                "value": "in multiple places."
              }
            ]
          }
        ]
      }
    ],
    "headings": {
      "2": [
        "Blockquote with breaks"
      ]
    }
  }
]